### ISSUE TITLE
Problem locating gtkgl-2.0 on Windows

### ISSUE THREAD

**Issue by pbrown12303** [mattn/go-gtk#331]
When installing go-gtk it complains that it cannot find gtkgl-2.0:

************  Building Go project: go-gtk  ************
  with GOPATH: C:\GoWorkspace\
>> Running: C:\Go\bin\go.exe install -v -gcflags "-N -l" github.com/mattn/go-gtk/...
github.com/mattn/go-gtk/gtkgl
github.com/mattn/go-gtk/gtksourceview
github.com/mattn/go-gtk/gtkspell
# pkg-config --cflags gtkgl-2.0
Package gtkgl-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `gtkgl-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'gtkgl-2.0' found
pkg-config: exit status 1

When I do pacman -Ss gtkgl what shows up is gtkglext which does not satisfy the dependency. A bit of research seems to indicate that the gtkgl-2.0 is obsolete ant gtkglext should be used instead. Any suggestions as to how to proceed?

**Comment by mattn** [comment]
No, if installed gtkgl package, you can get something flags for that with `pkg-config --cflags --libs gtkgl-2.0`
